0021-protocol-naming.adoc

```diff
@@ -0,0 +1,46 @@
+= Introducing a protocol naming scheme
+Thomas Eizinger <thomas@coblox.tech>; Daniel Karzel<daniel.karzel@coblox.tech>
+:toc:
+:revdate: 2020-01-20
+
+NOTE: Author: {authors} +
+Date: {revdate} +
+Tracking issue: https://github.com/comit-network/RFCs/issues/130[#130]
+
+== Proposal
+
+We propose the following protocol names for Alpha and Beta ledger:
+
+- `HAN` - HTCL for Asset on Native
```

Comment:
Perhaps, 'HTLC Native Asset' - HNA

Reply:
The tokens would become 'HTLC Custom Asset' - HCA

Reply:
`HAN` just reads and sounds nicer, that's why we changed the order. In the beginning we had `NAH` (which is like a fun version of "no" in Austrian dialect). HAN just makes it easy to pronounce.

Reply:
Nonetheless "HTLC for asset on native" sounds weird.

Reply:
Agree with @da-kami here. The abbreviation is massively more important than how you expand it. The expansion of `HAN` could also read:

(Atomic swap using) **H**tlcs of **A**ssets that are **N**ative to the ledger

HAN can be easily pronounced, is short and contains all the important information to differentiate it from other protocols.

At this stage, I would suggest for reviewers to make a counter proposal that is at least equally good (measured by all the aspects mentioned in the spike) if they are not happy with the result.

Reply:
> HAN just reads and sounds nicer, that's why we changed the order.

Reply:
I agree with @D4nte that "HTLC for asset on native" sounds rubbish but also with the others that 'HAN' is a better acronym.  "HTLC Asset Native" is fine though and satisfies both camps :)

Reply:
You are late to the party they already changed the proposal ;)